src/main/java/Duke.java

```diff
@@ -103,34 +107,35 @@ private static int getIndex(String input) {
         return index;
     }
 
-    private static Task getTodo(String input) {
+    private static Task getTodoDetails(String input) {
         String todoDescription = input.substring(4).trim();
+
         Task todo = new Todo(todoDescription);
-        lists[count] = todo;
+        list[count] = todo;
         count++;
 
         return todo;
     }
 
-    private static Task getDeadline(String input) {
+    private static Task getDeadlineDetails(String input) {
         int endIndex = input.indexOf("/");
         String deadlineDescription = input.substring(8, endIndex);
-        String deadlineDate = getDate(input);
+        String deadlineDate = getDateFromCommand(input);
```

Comment:
not too sure whether this is necessary

Reply:
Appreciate the effort you took to notice this. Since deadline requires two parameters (`description` and `by`), I am extracting those first similar to what I have done in `getEventDetails` below

Reply:
Appreciate the effort you took to notice this. Since deadline requires two parameters (description and by), I am extracting those first similar to what I have done in getEventDetails below

Reply:
Appreciate the effort you took to notice this. Since deadline requires two parameters (`description` and `by`), I am extracting those first similar to what I have done in `getEventDetails` below